cmd/oras/internal/option/packer.go

```diff
@@ -70,6 +72,22 @@ func (opts *Packer) ExportManifest(ctx context.Context, fetcher content.Fetcher,
 	return os.WriteFile(opts.ManifestExportPath, manifestBytes, 0666)
 }
 
+// ExportManifestDesc saves the descriptor of pushed manifest to a local file.
+func (opts *Packer) ExportManifestDesc(desc ocispec.Descriptor) error {
+	if opts.ManifestDescExportPath == "" {
+		return nil
+	}
+	manifestDescBytes, err := json.Marshal(ocispec.Descriptor{
+		MediaType: desc.MediaType,
+		Digest:    desc.Digest,
+		Size:      desc.Size,
```

Comment:
Can you confirm what is need in this descriptor? 

I saw in @shizhMSFT‘s [comment](https://github.com/oras-project/oras/issues/497#issuecomment-1216325804), the ` "containerimage.descriptor"` also contains annotation.